src/components/TextField/tests/TextField.test.tsx

```diff
@@ -1042,15 +1039,13 @@ describe('<TextField />', () => {
       );
       let connectedInteriorWrapper = textField
         .find(Connected)
-        .find('div')
-        .first();
+        .find('div.TextField');
 
       connectedInteriorWrapper.simulate('focus');
 
       connectedInteriorWrapper = textField
         .find(Connected)
-        .find('div')
-        .first();
+        .find('div.TextField');
 
       expect(connectedInteriorWrapper.hasClass('focus')).toBe(true);
```

Comment:
Same as in web we follow their best practice and avoid using classNames in tests. [see here](https://github.com/Shopify/web-foundation/blob/master/Best%20practices/React/Testing.md#implementation-details). `document.activeElement` would be more reliable here.

Reply:
The test is called `applies focus variant style onFocus`. Its whole purpose is to test for the `focus` class name. I was just getting it to work given the change in DOM output caused by this PR. The previous test called `sets focus to the <input /> onClick` checks for `document.activeElement` albeit from a different trigger.

I can ditch the whole test, if you'd like.

Reply:
Dan's guidance on asserting `document.activeElement` is probably a better approach here.